.github/workflows/genie-performance-test.yml

```diff
@@ -0,0 +1,44 @@
+name: genie-performance-test
+
+on:
+  schedule:
+    - cron:  '0 8 * * *'
+
+jobs:
+  probe-genie-endpoint:
+    
+    name: Genie private endpoint probe
+
+    runs-on: ubuntu-latest
+    
+    env:
+      SLACK_URL_SECRET: ${{ secrets.SLACK_URL_SECRET }}
+      SLACK_CHANNEL: backend-performance-status 
```

Comment:
nice! so cool! I love that it's easy to see in code where to sign up for this alert. One question: is this channel private? I can't seem to join - maybe ok if this channel is public?